### PR TITLE
Update filter.md

### PR DESCRIPTION
**Pull request by valerie-schneider**
Removed _Note that all the filters available on the products are also available on the published products._

Added on each paragraph "can be also used for published product"

### CODE REVIEW

File: content/rest-api/filter.md [hunk 1]
```diff
@@ -369,6 +384,10 @@ You can even combine several filters on the same attribute. The example below wi
 /api/rest/v1/products?search={"description":[{"operator":"NOT EMPTY","locale":"en_US","scope":"ecommerce"},{"operator":"EMPTY","locale":"fr_FR","scope":"ecommerce"}]}
 ```
 
+:::info
+Filtering on product values is also available for published products.
```
Comment (NolwennP) [inline content/rest-api/filter.md]: As it has no meaning to use this filter, let's remove this mention.

File: content/rest-api/filter.md [hunk 2]
```diff
@@ -381,6 +400,10 @@ is equivalent to
 /api/rest/v1/products?search={"description":[{"operator":"STARTS WITH","value":"Amazing","scope":"ecommerce"}],"short_description":[{"operator":"CONTAINS","value":"shoes","scope":"ecommerce"}]}&search_locale=en_US
 ```
 
+:::info
+`search_locale` query parameter is also available for the published products.
```
Comment (NolwennP) [inline content/rest-api/filter.md]:
```suggestion
This query parameter is also available for the published products.
```

File: content/rest-api/filter.md [hunk 3]
```diff
@@ -102,6 +108,9 @@ To get the products that are 100% complete on both the `en_US` and `fr_FR` local
 ```
 /api/rest/v1/products?search={"completeness":[{"operator":"GREATER OR EQUALS THAN ON ALL LOCALES","value":100,"locales":["en_US","fr_FR"],"scope":"ecommerce"}]}
 ```
+:::info
+Filtering on completness is also available for published products.
```
Comment (NolwennP) [inline content/rest-api/filter.md]:
```suggestion
Filtering on completeness is also available for published products.
```

File: content/rest-api/filter.md (hunk 1 above)
Comment (valerie-schneider) [reply content/rest-api/filter.md]: I'm sorry I don't understand why :/ In this case, the two mentions above has also no sense, no?

**Pull request merged**